utilities/convert_video_to_gxf_entities.py

```diff
@@ -1015,6 +1019,12 @@ def main():
     parser.add_argument("--width", required=True, type=int, help="Input width in pixels")
     parser.add_argument("--height", required=True, type=int, help="Input height in pixels")
     parser.add_argument("--channels", default=3, type=int, help="Channel count")
+    parser.add_argument(
+        "--duration",
+        default=-1,
+        type=int,
+        help="process only the first <duration> seconds of the video. Setting to -1 processes the entire video",
```

Comment:
Could we accept a `float` here for greater precision? I.e. allowing videos of less than 1 second.

Formatting nitpick:

```suggestion
        help="Process only the first <duration> seconds of the video. Setting to -1 processes the entire video",
```